content/docs/upgrading/upgrade.md

```diff
@@ -4,11 +4,67 @@ description = "Upgrading your deployment to a later version of Kubeflow"
 weight = 30
 +++
 
-Until version 1.0, Kubeflow makes no promises of backwards compatibility or 
-upgradeability. There is **no clean upgrade path** to the latest version of 
-Kubeflow ({{% kf-latest-version %}}).
+Upgrading your Kubeflow deployment is supported if your deployment is v0.7.0 or later.
 
-Nonetheless, here are some instructions for updating your deployment:
+Prerequisites:
+
+* Download the latest kfctl binary from the
+  [Kubeflow releases page](https://github.com/kubeflow/kubeflow/releases/tag/{{% kf-latest-version %}}).
+
+* Ensure that your kubeflow namespace is annotated with the following label:
+```
+  labels:
+    control-plane: kubeflow
+```
+* You must have a local Kubeflow application directory matching your current
```

Comment:
We should add some context here.

If they used kfctl to install Kubeflow, then they already have a Kubeflow application directory. The installation guides use the environment variable `${KF_DIR}` to refer to this directory.

If they didn't use kfctl (for example, they used the [GCP Kubeflow deployment web app](/docs/gke/deploy/deploy-ui/)) then we need to tell them how to create the local deployment directory. That info would be useful in other scenarios too, so I think it'd be good to create a separate page explaining how to do this. The page could be in the [other guides](https://deploy-preview-1455--competent-brattain-de2d6d.netlify.com/docs/other-guides/) section.

Reply:
Added more context. We don't have a page for generating a local deployment directory yet; once we have that we can add the link here.

Reply:
We do need to give them some idea of how to create the local deployment directory. What should be in it and where can they get the files from?

Reply:
Alternatively we can say that upgrades are only supported if your initial deployment is done through kfctl.

Generating the local deployment directory isn't very straightforward. The user needs to do something like:
- Create a local directory with a name matching "deployment name"
- Follow all of the preparation steps in https://www.kubeflow.org/docs/gke/deploy/deploy-cli/#prepare-your-environment
- Run kfctl build -f $KFDEF, where $KDFEF is:
    - If using IAP, use https://github.com/kubeflow/manifests/blob/master/kfdef/kfctl_gcp_iap.yaml
    - If using username/password, use https://github.com/kubeflow/manifests/blob/master/kfdef/kfctl_gcp_basic_auth.yaml

Reply:
This seems a little error-prone. I would prefer to just say that upgrading is only supported if you initially deployed with kfctl, and consider either (1) creating a script that automatically generates the local config directory or (2) implement upgrades in the UI.

Reply:
Added some instructions - let me know if this is clear enough.